public/js/controllers/game.js

```diff
@@ -201,5 +213,61 @@ angular.module('mean.system')
         game.joinGame();
       }
 
+  $scope.inviteUsers = function () {
+  $scope.hideDiv = true;
+    if($scope.inviteList.length === game.playerMaxLimit - 1 ){
+          $('#modalView').modal('show');
+        return;
     }
-  ]);
+    if($scope.inviteList.includes($scope.email)){
+                $('#modalView1').modal('show');
+        return;
+    }
+    
+      $http({
```

Comment:
Abstract this functionality to services. Controllers should not be used to make `http` requests rather only to bind data to the Scope.